js/redirect.js

```diff
@@ -0,0 +1,27 @@
+var matches = /Firefox\/(\d+)/.exec(navigator.userAgent);
+	
+// if using Fx
+if (matches !== null && matches.length > 1) {
+	var version = parseInt(matches[1], 10);
+
+	// if using a version with about:home (4 or greater), redirect them there
+	if(version >= 4) {
+		document.location.href = 'about:home';
+	}
+
+	// make sure platform is upgradeable (not on PPC or OS X 10.0 - 10.4)
+	if ( (/(PPC|Mac OS X 10.[0-4])/.test(navigator.userAgent) === false) && (version < 29) ) {
+		// display appropriate message
+		if (version === 3) {
+			$('#fx3').show();
+		} else if (version === 4) {
+				$('#fx4').show();
+		} else {
+				$('#fxother').show();
+		}
+	
+	}
```

Comment:
I don't think it makes sense to duplicate the messaging logic from `ffx36.js` here. Also, since this file is loaded in the header (before jQuery), use of the `$` global may result in errors (as jQuery may not yet be loaded).

I'd suggest this file only handle redirects. Any page-specific behavior should stay in `ffx36.js`.